Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -350,36 +373,14 @@ internal static void UpdateLevelKind(MatchExpression matchExpression)
             return result;
         }
 
-        public override AnalyzeContext InitializeGlobalContextFromOptions(AnalyzeOptions options, ref AnalyzeContext context)
```

Comment:
>InitializeGlobalContextFromOptions [](http://example.com/codeflow?start=39&length=34)

It looks like much change here, compare to other small; changes, could you explain a bit the change here

Reply:
There should be no functionality changes AT ALL. This is only to make the compiler happy to have things in the order it wants.

Reply:
Yes, sorry, this was a really messy one. properties not before fields. Constructor not before fields. public not before static.
I pretty much had to completely rearrange to satisfy all the different conditions.

Reply:
properties not before fields. Constructor not before fields. public not before static. --- got it.